src/plugin/settings/settings_model.h

```diff
@@ -78,7 +78,7 @@ static const std::string kEngineSettingsModel = /*suppress newline*/ 1 + R"json(
                     "description": "Cloud bucket access key ID",
                     "defaultValue": "",
                     "validationErrorMessage": "Access key ID must be >=16 alphanumeric characters (uppercase or 2-7).",
-                    "validationRegex": "^[A-Z2-7]{16,128}$"
+                    "validationRegex": "^[A-Z1-9]{16,128}$"
```

Comment:
When I looked into this, I found sources saying the access key ID is base32 encoded, which doesn't use `0`, `1`, `8` or `9`.

Reply:
I think it can support more. Here is some from an old blog from AWS https://aws.amazon.com/blogs/security/a-safer-way-to-distribute-aws-credentials-to-ec2/. I have also verified credentials from Lyve cloud that include the digits 0,1,8 and 9.